dist/gdb/Makefile.mingw

```diff
@@ -15,6 +15,7 @@ gdbtargets=$(addprefix ${targetdir}/bin/gdb-, ${targets})
 packageparts=${targetdir}/bin/lib ${targetdir}/bin/lib ${targetdir}/bin/libiconv-2.dll ${targetdir}/bin/python27.dll ${targetdir}/bin/libexpat-1.dll
 arch=`uname -sm | sed 's/ /-/g' | tr A-Z a-z`
 packagename=sailfish-gdb-${version}-windows.7z
+mingwbin=/c/MinGW/bin
```

Comment:
No way to compute this somehow?

Reply:
I was thinking this myself as well, but could figure out an easy way. Of course we could hack something together, like check all directories in PATH if they happen to contain the files we want, but I don't really think that's a good idea. I can still try to think of something while waiting for Jenkins to build...

Reply:
maybe... isn't that the dir where gcc executable exists? `mingwbin=$(dirname "$(which gcc-exact-name)")`?

Reply:
Yes. I'll try that after I have confirmed that this really fixes the issue.